second.cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+struct OrderedDoubles
+{
+    int first = 1;
+    int second = 1;
+};
+bool compareOrderedDoubles(OrderedDoubles firstNum, OrderedDoubles secondNum)
```

Comment:
Typically when comparing two things we have 3 cases `left is bigger than right`, `left is equal to right`, `left is smaller than right`. `bool` however is only capable of being `true` or `false`, meaning that we cannot cover the 3 possible cases mentioned. The typical approach for comparison in the majority of programming languages is to `return a negative value (e.g. -1) if left is smaller than right`, `return a zero value if left is equal to right` and `return a positive value (e.g. +1) if left is bigger than right` :))